docs/data/xrootd/install-standalone.md

```diff
@@ -118,6 +108,8 @@ To set up XRootD in multi-user mode, install the `xrootd-multiuser` package:
 root@xrootd-standalone # yum install xrootd-multiuser
 ```
 
+If you are using XRootD-Multiuser with a VOMS FQAN, you need XRootD 5.5.0 or greater.
```

Comment:
This is a little hidden, let's turn it into an admonition. Pick your favorite one: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types